os/os.proto

```diff
@@ -65,14 +65,23 @@ service OS {
   //                              <-- [Validated|InstallError]
```

Comment:
with line 19 please update to:

// Dual Supervisor Target is supported, where the above process is executed once
// for each Supervisor or once for the whole Cluster.

Reply:
thanks for review, I have updated to this.